src/Miunie.Core/Strings.resx

```diff
@@ -158,6 +158,22 @@ Source code available at: {1}</value>
     <value>**ABOUT THIS SERVER**</value>
     <comment>No formatting options.</comment>
   </data>
+  <data name="HELP_EXAMPLE_EMPTY" xml:space="preserve">
+    <value>No examples were provided.</value>
```

Comment:
We should probably be more creative and utilize variance with `{{OR}}`.
What about something like:
```
You'll figure it out.{{OR}}
You know what? I've never used this command.{{OR}}
Yeah, you're on your own with this one, buddy.
```

Or similar. 🙂